### DISCUSSION TITLE
Provider References Field

### DISCUSSION THREAD

**Discussion by truffledonal** [CMSgov/price-transparency-guide#650] [Q&A]
Healthcare newbie here.

Reaching out to gain a better understanding of what exactly the `provider_references` field means and what is is used for. 

As I understand it, `provider_references` is a list of provider reference objects. These providers are then referenced in the `negotiated_rates` array. When providers are referenced for a negotiated rate, **does it means that this specific negotiated rate entry applies only for those providers?** I.e., if you got this service done at one of these providers, under this plan the negotiated rate is x? 

Based on this understanding, I am confused why `provider_references` would then be marked an optional field in the schema. If it was missing, then none of the negotiated rates would be valid for any providers, which to me does not sound too useful. **If a negotiated rate has no listed `provider_references`, how do we know for which providers that negotiated rate is valid?** 

I ask because I am facing issues finding negotiated rates for the specific providers (all in-network) my clients were treated at, particularly being unable to find these providers. 

If anyone can shed light on my understanding that would be great!

**Comment by AshleyOboe** [comment]
@truffledonal - I think your understanding is correct. 
The provider references object can either be in a stand-alone object (key-value section) in the in-network file or embedded in the negotiate rate object, so I think that's why it is listed as optional in both places.  

In either case, the technical requirements are the health plans list individual NPIs associated with an entity ID.  I can't find it right now in the guidance but I believe I read that if the rate applies to all NPIs in an organization, they are allowed to use an NPI of "000000000".  Don't quote me on that, but we do see values of "000000000" and "0" frequently in the data in the NPI field.   Also, plans are permitted to use an NPI in place of a tax ID if the provider uses a SSN for contracts.

If you want to message me I'm happy to see if I can find the NPIs you're searching for in our data.

**Reply by truffledonal** [reply]
Thank you for the swift reply! That definitely clears up a lot - I am certainly interested in the "catch-all" default NPI value, however the plans relevant to me do not appear to have an all zeros NPI number. 

I do already have the NPI numbers, but they are not present in the downloaded plan MRF for the respective plan my clients have. For example, for one of the health plans, these are the only NPI numbers available (shown below). These are only 4 providers, and my client who uses this plan has seen in network providers with NPI numbers not listed below. I figure `provider_group_id` is referring to the fact that it is the first (and only, as it happens) provider group in the `provider_references` list. 

`"provider_references": [{"provider_groups": [{"npi": [1932245974, 1528566890, 1790933828, 1588982060], "tin": {"type": "ein", "value": "208325555"}}], "provider_group_id": 0}]`

**How do I find negotiated rates for in network providers that are not included in the `provider_references` field?**

**Comment by shaselton-usds** [comment]
@AshleyOboe A value of `0` in the NPI field indicates that the NPis are unknown because negotiates are made at the TIN level. You can see that answer in the [technical clarification, number 35](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification)

@truffledonal You would find the negotiated rates for the `provider_groups` by matching the `provider_group_id` (in this case `0`), to the values found in the `provider_references` array found in the specific `negotiated_rates` object. [Here is an example.](https://github.com/CMSgov/price-transparency-guide/blob/31ec1821956cee7b5ac02780a2246bd09d4effd1/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L87)

**Reply by truffledonal** [reply]
That makes sense, my problem however is that I cannot find the relevant in-network provider NPI in the `provider_references` array for this plan. In the above example, those are the only NPI's present in the entire plan file. My client, who has that insurance plan, saw an in-network provider whose NPI is not present in that file's `provider_references`. How can I validate the negotiated rate for this provider if their NPI is not in the `provider_references` array?

**Reply by shaselton-usds** [reply]
Ah, I see. You are starting with NPI number and then are trying to locate it within the file vs seeing a NPI in the `provider_references` and then locating that NPI with associated services.

Though I cannot speak directly to why you aren't finding the provider's NPI in the file, there might be some possible reasons. First, if a contract is made at the NPI type 2 level (the organization vs the individual), it is possible that the single provider's NPI just isn't available at the contracting level and thus the type 2 NPI was reported. Also, as you noted earlier with the `0` value; there are times when negotiations happen even further up at the TIN level without any understanding of NPIs under neither them, so that individual NPI context would be difficult to report on. Lastly, as a sanity check, there can be quite a few files out there so double checking that the correct file being evaluated is always recommended.
